<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WayPointItemWgt</class>
 <widget class="QWidget" name="WayPointItemWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>137</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="edgeWgt" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>动作：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="actionCB">
          <item>
           <property name="text">
            <string>航点</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>变高</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>起飞</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>盘旋</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>返航</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>跳转</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="actSWgt">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="wpPage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>纬度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="latLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>经度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lngLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>海拔：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="altLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="chgAltPage">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>高：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="flyPage">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_3"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="cyclePage">
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_4"/>
          </item>
          <item row="4" column="2">
           <widget class="QComboBox" name="comboBox_5"/>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBox_6"/>
          </item>
          <item row="5" column="2">
           <widget class="QComboBox" name="comboBox_7"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="returnPage">
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="jumpPage">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>跳转至：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_8"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
